auth/src/main/java/com/firebase/ui/auth/ui/ActivityHelper.java

```diff
@@ -106,4 +107,8 @@ public static Intent createBaseIntent(
                 .putExtra(ExtraConstants.EXTRA_FLOW_PARAMS,
                         checkNotNull(flowParams, "flowParams cannot be null"));
     }
+
+    public SmartLock getSmartLockInstance(AppCompatBase activity, String tag) {
+        return SmartLock.getInstance(activity, tag);
```

Comment:
Does this method do anything besides pass through?  Seems like we could just use the static method.